src/compiler/config/load-config.ts

```diff
@@ -15,7 +15,22 @@ import { validateConfig } from './validate-config';
 import { validateTsConfig } from '../sys/typescript/typescript-config';
 import ts from 'typescript';
 
-export const loadConfig = async (init: LoadConfigInit = {}) => {
+/**
+ * Load and validate a configuration to use throughout the lifetime of any Stencil task (build, test, etc.).
+ *
+ * Users can provide configurations multiple ways simultaneously:
+ * - as an object of the `init` argument to this function
+ * - through a path to a configuration file that exists on disk
+ *
+ * In the case of both being present, the two configurations will be merged. The fields former will take precedence
```

Comment:
```suggestion
 * In the case of both being present, the two configurations will be merged. The fields of the former will take precedence
```

^^ little wording suggestion